dell/src/test/java/org/apache/iceberg/dell/mock/ecs/ObjectData.java

```diff
@@ -20,18 +20,25 @@
 
 import com.emc.object.Range;
 import com.emc.object.s3.S3ObjectMetadata;
-import com.emc.object.shadow.org.apache.commons.codec.digest.DigestUtils;
```

Comment:
isn't that using a shaded version, meaning that we wouldn't have to change this file?

Reply:
I replaced the code for native Java 8 code, then that shaded version is not going to be needed.

Reply:
I have replaced with native Java code